data/korath/korath ships.txt

```diff
@@ -15,62 +16,70 @@ ship "Korath Dredger"
 	attributes
 		category "Heavy Freighter"
 		"cost" 29370000
-		"shields" 39000
-		"hull" 16000
-		"required crew" 201
-		"bunks" 395
+		"shields" 27000
+		"hull" 22000
```

Comment:
Reasoning?

Reply:
Looking back at it, 22k hull seems a bit high compared to the Raiders given the mass ratio.
This might be a bit too low, though.
```suggestion
		"shields" 33000
		"hull" 18000
```